src/Agent/NewRelic/Agent/Core/NewRelic.Agent.Core.Metric/ApiSupportabilityMetricCounters.cs

```diff
@@ -37,7 +37,8 @@ public enum ApiMethod
         SpanAddCustomAttribute = 23,
         InsertDistributedTraceHeaders = 24,
         AcceptDistributedTraceHeaders = 25,
-        SpanSetName = 26
+        SpanSetName = 26,
+        ErrorFingerprintingCallback
```

Comment:
```suggestion
        ErrorFingerprintingCallback = 27
```

Reply:
Yeah, probably a good idea to keep with the convention.